### DISCUSSION TITLE
how can i load po file in koreader

### DISCUSSION THREAD

**Discussion by Monirzadeh** [koreader/koreader#10016] [Q&A] [answered]
For now, I work on translate Persian language for KOReader that not exist in language list right now.
I think it is getting me a better idea if I can copy `po` file in specific folder and KOReader load that language in the list.
So I can see all my translation on my device that can get me better overview of final result with day to day use.
Is it possible to do this?
I am not familiar so much with workflow of adding new translation to KOReader and not find useful information in wiki.
For now, I am translating [here](https://hosted.weblate.org/translate/koreader/koreader). But if I can load `po` file in my day-to-day device, it will get me better context for translation.

**Comment by Frenzie** [comment, marked as answer]
To show it on the device it should suffice to uncomment this line and overwrite the PO file. It was commented so as not to give the impression that a few translated lines constitute a working translation, so I'll change that now.
https://github.com/koreader/koreader/blob/d1cd2a1c8d9040ed9edee4fee5ec5b34b05cd9a1/frontend/ui/language.lua#L151

![Screenshot_20230110_212027](https://user-images.githubusercontent.com/202757/211653795-6beb192c-8b80-4001-b8fa-fca921f82bc0.png)
